client/components/date-range/test/index.js

```diff
@@ -775,93 +682,48 @@ describe( 'DateRange', () => {
-				dateToLocaleString( '05/28/2018' )
-			);
-		} );
-	} );
+			await userEvent.click( screen.getByLabelText( 'Select date range' ) );
 
-	describe( 'Render props', () => {
-		test( 'should allow for render prop to overide trigger render', () => {
-			const spyComponent = jest.fn();
+			const fromEl = screen.getByLabelText( 'From' );
+			const toEl = screen.getByLabelText( 'To' );
 
-			shallow(
-				<DateRange translate={ translate } moment={ moment } renderTrigger={ spyComponent } />
-			);
+			await userEvent.click( fromEl );
+			await userEvent.clear( fromEl );
+			await userEvent.keyboard( '04/13/2018' );
+			await userEvent.click( toEl );
+			await userEvent.clear( toEl );
+			await userEvent.keyboard( '05/13/2018' );
+			// force a blur out of `toEl`
+			await userEvent.click( fromEl );
 
-			const props = spyComponent.mock.calls[ 0 ][ 0 ];
-			const propKeys = Object.keys( props ).sort();
-
-			expect( spyComponent ).toHaveBeenCalledTimes( 1 );
-
-			expect( propKeys ).toEqual(
-				[
-					'startDate',
-					'endDate',
-					'startDateText',
-					'endDateText',
-					'isCompact',
-					'buttonRef',
-					'onTriggerClick',
-					'onClearClick',
-					'triggerText',
-					'showClearBtn',
-				].sort()
+			expect( screen.getByLabelText( 'Fri, Apr 13, 2018 12:00 PM' ) ).toHaveAttribute(
+				'aria-selected',
+				'true'
 			);
-		} );
-
-		test( 'should allow for render prop to overide header render', () => {
-			const spyComponent = jest.fn();
-
-			shallow(
-				<DateRange translate={ translate } moment={ moment } renderHeader={ spyComponent } />
+			expect( screen.getByLabelText( 'Sun, May 13, 2018 12:00 PM' ) ).toHaveAttribute(
+				'aria-selected',
+				'true'
 			);
 
-			const props = spyComponent.mock.calls[ 0 ][ 0 ];
-			const propKeys = Object.keys( props ).sort();
-
-			expect( spyComponent ).toHaveBeenCalledTimes( 1 );
-
-			expect( propKeys ).toEqual( [ 'onApplyClick', 'onCancelClick' ].sort() );
-		} );
-
-		test( 'should allow for render prop to overide inputs render', () => {
-			const spyComponent = jest.fn();
+			const resetBtn = screen.getByLabelText( 'Reset selected dates' );
+			await userEvent.click( resetBtn );
 
-			shallow(
-				<DateRange translate={ translate } moment={ moment } renderInputs={ spyComponent } />
+			expect( fromEl.value ).toEqual( '04/28/2018' );
+			expect( toEl.value ).toEqual( '05/28/2018' );
+			expect( screen.getByLabelText( 'Sat, Apr 28, 2018 12:00 PM' ) ).toHaveAttribute(
+				'aria-selected',
+				'true'
 			);
-
-			const props = spyComponent.mock.calls[ 0 ][ 0 ];
-			const propKeys = Object.keys( props ).sort();
-
-			expect( spyComponent ).toHaveBeenCalledTimes( 1 );
-
-			expect( propKeys ).toEqual(
-				[ 'startDateValue', 'endDateValue', 'onInputChange', 'onInputBlur', 'onInputFocus' ].sort()
+			expect( screen.getByLabelText( 'Mon, May 28, 2018 12:00 PM' ) ).toHaveAttribute(
+				'aria-selected',
+				'true'
 			);
 		} );
 	} );
 
+	// TODO Other tests?
```

Comment:
I will remove this comment soon. Any additional tests - if any - will be added in a follow-up PR.